test/gerkin/tests/features/data_list.feature

```diff
@@ -0,0 +1,20 @@
+Feature: API endopoint allowing users to search multiple records in the Digital Registries database.
+  Request endpoint: GET /data/{registryname}/{versionnumber}/
+
+  Background:
+    Given The user wants to search for "John Helmut" in the Digital Registries database
+
+  Scenario: The user receives a list with all records including "John Helmut" in the Digital Registries database
+    When The user triggers an action to search "John Helmut" in the database
```

Comment:
Not enough details in these scenarios, there should be information if the record "John Helmut" exists or not - based on that information the result would be either empty or it would include searched data.